frontend/src/console/components/FooterPreview/FooterPreview.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
```

Comment:
@Agrendalath Footer preview page looks a bit off.
- The spacing between the link items looks too large
- The `About` link is not aligned with the logo and the other links
- The OpenEdX logo needs a bit more of vertical padding.
- There's no link hover effect (on the platform is `underline` if I'm not mistaken).